db/migrate/20151123235600_add_purchase_type_to_gsa18f_procurements.rb

```diff
@@ -0,0 +1,11 @@
+class AddPurchaseTypeToGsa18fProcurements < ActiveRecord::Migration
+  def change
+    add_column :gsa18f_procurements, :purchase_type, :integer
+
+    execute <<-SQL
+      UPDATE gsa18f_procurements SET purchase_type = 0 WHERE purchase_type IS NULL;
+      SQL
```

Comment:
Hmmmm, that method really needs some documentation :unamused: I'd simplify all of this to:

```ruby
add_column :gsa18f_procurements, :purchase_type, :integer, default: 0, null: false
```